source/databases-collections.txt

```diff
@@ -34,38 +34,35 @@ use the left navigation to:
 Databases and Collections
 -------------------------
 
-When you expand an active connection, |vsce| shows the databases in that
-deployment. Click a database to view the collections it contains.
+When you expand an active connection, |vsce| shows the databases in
+that deployment. Click a database to view the collections it contains.
 
 Click a collection to view its documents, schema, and indexes.
 
+.. include:: /includes/admonitions/close-all-docs-on-exit.rst
+
 View Collection Documents and Schema
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Documents
 ``````````
 
-When you expand a collection, |vsce| displays that collection's 
-document count next to the :guilabel:`Documents` label in the 
+When you expand a collection, |vsce| displays that collection's
```

Comment:
|vsce| shows the number of documents next to the :guilabel:`Documents` label in the ...

Reply:
Agreed. Fixed.